lib/fathead/htmlref/cover/tags.txt

```diff
@@ -0,0 +1,142 @@
+a
+abbr
+address
+area
+article
+aside
+audio
+b
+base
+bdi
+bdo
+blockquote
+body
+br
+button
+button type=submit
```

Comment:
Looks like we just have "button" (https://github.com/sriranganathan/zeroclickinfo-fathead/blob/34104284f62b55591c3c09ae2e7eaa9fc6fab647/lib/fathead/htmlref/output.txt#L95). You can remove `button type=submit` and the others. Otherwise we're going to get errors about the output.txt missing articles we expected to have, according to the coverage data 👍